### PR TITLE
C code for two issues assigned to me

### PR DESCRIPTION
**Pull request by TechSpiritSS**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

I wrote C code for issue #2865 and issue #2861 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

issue #2865 
issue #2861

### CODE REVIEW

**Comment by TechSpiritSS** [comment]
There was nothing to do in readme.md so I kept that as it is

**Comment by TechSpiritSS** [comment]
This was my first pull here; I'll try to provide each issue with its own PR instead of this combined one

**Comment by RaiyanMahin** [comment]
Don't push more than one code file into the same PR. Break down your PRs into separate PRs so that it becomes easy to review.

File: C/other/LCM.c [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+By Sidharth Sethi
+github - TechSpiritSS
```
Comment (RaiyanMahin) [inline C/other/LCM.c:3]: remove your name

File: C/other/LCM.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    //Loop till both aren't the same
+    while(a!=b)
+    {
+        if(a<b)
+            a+=a1;
+        else
+            b+=b1;
+    }
+    printf("LCM of %d and %d is %d", a1,b1,a);
+}
```
Comment (RaiyanMahin) [inline C/other/LCM.c:23]: add a new line at EOF

**Review by RaiyanMahin — CHANGES_REQUESTED** [review]

File: C/other/LCM.c [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/*
+By Sidharth Sethi
+github - TechSpiritSS
+*/
+#include <stdio.h>
```
Comment (RaiyanMahin) [inline C/other/LCM.c:5]: use main function ( ) for driven the code

File: C/other/LCM.c [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    b1=b;
+    //Loop till both aren't the same
+    while(a!=b)
+    {
+        if(a<b)
+            a+=a1;
+        else
+            b+=b1;
+    }
+    printf("LCM of %d and %d is %d", a1,b1,a);
```
Comment (RaiyanMahin) [inline C/other/LCM.c:22]: add time complexity and space complexity of your code

File: C/other/LCM.c [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    a1=a;
+    b1=b;
+    //Loop till both aren't the same
+    while(a!=b)
+    {
+        if(a<b)
+            a+=a1;
+        else
+            b+=b1;
+    }
```
Comment (RaiyanMahin) [inline C/other/LCM.c:21]: add sample I/O

File: C/other/LCM.c [hunk 6]
```diff
@@ -0,0 +1,23 @@
+/*
```
Comment (RaiyanMahin) [inline C/other/LCM.c:1]: add a beginner friendly description

**Review by RaiyanMahin — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
Check some merged PR's for better understanding

**Comment by atarax665** [comment]
@TechSpiritSS Are you still working on this?

**Pull request closed without merging**